src/main/components/Browse/Browse.js

```diff
@@ -72,51 +70,16 @@ class Browse extends Component {
       true
     );
 
-  handleFilterChange = (dashboardAnalyses, name) => value => {
-    this.setState(
-      {
-        chosenFilters: {
-          ...this.state.chosenFilters,
-          [name]: value
-        }
-      },
-      () => {
-        const filters = this.state.chosenFilters;
-        let analyses = dashboardAnalyses;
-        analyses = analyses.filter(analysis =>
-          this.isWithinFilter(analysis, filters)
-        );
-        this.setState({ analyses: analyses });
-      }
-    );
+  handleFilterChange = name => value => {
+    const filters = {
+      ...this.state.chosenFilters,
+      [name]: value
+    };
+    this.setState({ chosenFilters: filters });
   };
 
-  handleOptions = (currentAnalyses, filter) =>
-    // get the options that pertain to the current filter type
-    currentAnalyses
-      .map(analysis => analysis[filter])
-      // if filter type is libraryId or sampleId, expand arrays into individual choices
-      .reduce(
-        (options, filter) =>
-          Array.isArray(filter)
-            ? [...options, ...filter]
-            : [...options, filter],
-        []
-      )
-      // filter out duplicates
-      .reduce(
-        (options, filter) =>
-          options.indexOf(filter) === -1 ? [...options, filter] : options,
-        []
-      )
-      // return options in a react-select friendly format
-      .map(option => {
-        return { label: option, value: option };
-      });
-
-  clearFilters = () => {
-    this.setState({
-      analyses: null,
+  resetFiltering = (closeDrawer = false) => {
+    let newState = {
```

Comment:
I would opt for this design instead, as I think it's a little cleaner:
```
getInitialFilterState = () => ({
      chosenFilters: {
        title: null,
        description: null,
        jiraId: null,
        libraryIds: null,
        sampleIds: null,
        project: null
      }
    })


handleAnalysisClick = () => this.setState({ ...getInitialFilterState(), isOpen: false,  })

onClose={() => this.setState({ ...getInitialFilterState(), isOpen: false })}
```
You can even use it in the constructor :)

```
    this.state = {
      ...getInitialFilterState(),
      isOpen: true
    };
```